src/ui/server/controllers/insurance/eligibility/uk-goods-or-services/index.ts

```diff
@@ -21,6 +21,7 @@ export const TEMPLATE = TEMPLATES.INSURANCE.ELIGIBILITY.UK_GOODS_OR_SERVICES;
 export const get = (req: Request, res: Response) =>
   res.render(TEMPLATE, {
     ...singleInputPageVariables({ ...PAGE_VARIABLES, BACK_LINK: req.headers.referer }),
+    user: req.session.user,
```

Comment:
just a sanity check:
a) do we need all properties from the req.session.user?
b) do we expect all properties to be available? or do we need to check `req.session?.user`?

Reply:
Good shout - we only really need the first and last name - will create a function to only return these properties if they exist.

BTW this is currently using the same approach as DTFS so might want to update it there as well

Reply:
cool. happy for this to be a separate PR if you want